data/pigui/modules/station-view/04-shipMarket.lua

```diff
@@ -215,69 +229,99 @@ function FormatAndCompareShips:compare_and_draw_column(desc, str, a, b)
 end
 
 function FormatAndCompareShips:draw_hyperdrive_cell(desc)
-	local hyperdrive_str = self.def.hyperdriveClass > 0 and
-		Equipment.hyperspace["hyperdrive_" .. self.def.hyperdriveClass]:GetName() or l.NONE
 
-	self:compare_and_draw_column( desc, hyperdrive_str, self.def.hyperdriveClass, self.b.def.hyperdriveClass )
+	local function fmt( v )
+		return v > 0 and
+			Equipment.hyperspace["hyperdrive_" .. v]:GetName() or l.NONE
+	end
+
+	self:compare_and_draw_column( desc, self.def.hyperdriveClass, self.b.def.hyperdriveClass, fmt )
 end
 
 function FormatAndCompareShips:get_value(key)
-	v = self[key]
+	local v = self[key]
 	if nil == v then
 		v = self.def[key]
 	end
 	return v
 end
 
 function FormatAndCompareShips:draw_tonnage_cell(desc, key)
-	self:compare_and_draw_column( desc, Format.MassTonnes(self:get_value(key)), self:get_value(key), self.b:get_value(key) )
+	self:compare_and_draw_column( desc, self:get_value(key), self.b:get_value(key), Format.MassTonnes )
 end
 
-function FormatAndCompareShips:draw_accel_cell(desc, thrustKey, massKey, multiplier)
+function FormatAndCompareShips:draw_accel_cell(desc, thrustKey, massKey, multiplier, inverse)
 	local accelA = multiplier * self.def.linearThrust[thrustKey] / (-9.81*1000*(self:get_value(massKey)))
 	local accelB = multiplier * self.b.def.linearThrust[thrustKey] / (-9.81*1000*(self.b:get_value(massKey)))
-	self:compare_and_draw_column( desc, Format.AccelG(accelA), accelA, accelB )
+	if inverse then
+		accelA = -accelA
+		accelB = -accelB
+	end
+	self:compare_and_draw_column( desc, accelA, accelB, Format.AccelG )
```

Comment:
As mentioned on IRC, this should be `accelA = self.def.linearThrust[thrustKey] / (9.8106 * 1000 * self:get_value(massKey))` with no `multiplier` or `inverse` terms. All accelerations are expressed with positive values, and the current code in the PR performs two inversions of that positive value to render the acceleration value.

Reply:
Sorry for wasting your time there, I really didn't think about that very much :-(  Still had a slightly sore head from the night before when I did this!